library/include/borealis/swkbd.hpp

```diff
@@ -22,14 +22,48 @@
 #include <functional>
 #include <string>
 
+#ifdef __SWITCH__
+#include <switch.h>
+#endif
+
 namespace brls
 {
 
+enum keyboard_disable_bitmask
```

Comment:
The point of making an abstraction is that you don't need to define the values twice - I would have only used the values after the #else in the enum, and then make another function to translate from the abstracted enum to the swkbd one, only for Switch.

Reply:
Also the name doesn't match the convention: `KeyboardKeyDisableBitmask` would fit better